cmd/kubeadm/app/cmd/phases/init/uploadcerts.go

```diff
@@ -70,5 +69,8 @@ func runUploadCerts(c workflow.RunData) error {
 	if err := copycerts.UploadCerts(client, data.Cfg(), data.CertificateKey()); err != nil {
 		return errors.Wrap(err, "error uploading certs")
 	}
+	if !data.SkipCertificateKeyPrint() {
```

Comment:
Can you add `SkipCertificateKeyPrint` flag to this phase?

Reply:
i was thinking about this today.
makes sense!

Reply:
updated.